docs/use_cases/user_gets_revoked.md

```diff
@@ -0,0 +1,18 @@
+# User gets revoked from a project #
+
+This happens whenever an user gets its permissions over the repository revoked. This use case has a side case in where an user might not be able to see any project in the dashboard due to lack of permissions to any project.
+
+## Assumptions ##
+
+- User logged in.
+- User at dashboard.
+
+## Steps ##
+
+1. If the user has no create permissions, and there're no projects, the dashboard might show an user-friendly UI where it gets informed that he has no access to any projects on its scope: _"If you think you should be seeing projects here, contact your GitHub administrator and request the appropriate permissions."_.
```

Comment:
What's a `create permission`, maybe should be rephrased?

Reply:
I was thinking in something like permissions on a file system, `read|write|execution` permissions. Is that the correct way to write it?

Reply:
Permissions here are 100% based on github. I think they have two: read/write
We shouldn't store any permission information though, we can just check github to see what permissions they have so we don't have to worry about enforcing permissions. We just have to check permissions to know what is the proper message is to display.

Reply:
So may I rephrase it like: 
> If the user has no write permissions over the organization, and...